src/helpers.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+use Weglot\Client\Client;
+use Weglot\Client\Endpoint\Languages;
+use Weglot\Client\Api\LanguageEntry;
+use Illuminate\Support\Facades\Request;
+
+if (! function_exists('currentLocale')) {
```

Comment:
Prefix with **weglot**. The function seems necessary so it must absolutely exist even if there is a conflict